spec/editors_draft.html

```diff
@@ -3298,6 +3298,231 @@ <h2>Evaluation Semantics</h2>
+        	and the parser emits a blank node <var>B</var> for occurrences of <var>id</var> outside of composite values,
+        	then the rewritten blank node label <var>b</var> MUST be a lexical form that maps to <var>B</var> in the lexical-to-value mapping (that is, <i><a>bnl2bn</a></i>(<var>id</var>) = <var>B</var>).</li>
+      </ul>
+
+      <p>
+        Here, "loading" is any single process by which composite values enter a SPARQL system or RDF Dataset. This includes (but is not limited to):
+      </p>
+
+      <ul>
+        <li>Parsing literal values in a [[SPARQL12-QUERY]] or [[SPARQL12-UPDATE]] string</li>
+        <li>The loading of RDF data via a single [[SPARQL12-GRAPH-STORE-PROTOCOL]] request</li>
+        <li>The loading of RDF data via [[SPARQL12-UPDATE]] `LOAD` operation</li>
+        <li>The loading of SPARQL Results via [[SPARQL11-FEDERATED-QUERY]]</li>
+        <li>The loading of RDF data via any single [[SPARQL11-QUERY]] `FROM` or `FROM NAMED` clause in systems <a data-cite="SPARQL11-SERVICE-DESCRIPTION#sd-dereferencesuris">supporting the dereferencing and loading of RDF data in the construction of the dataset</a></li>
+        <li>Importing of any RDF data to the underlying RDF Dataset (via means other than [[SPARQL11-UPDATE]] or [[SPARQL12-GRAPH-STORE-PROTOCOL]])</li>
+      </ul>
+
+      <p>Loading the Turtle in <a href="#load-renaming-blank-example-simple"></a>, data with `_:b0` in a <a>cdt:List literal</a> MUST result in an (internal) dataset with a single triple whose object is a <a>cdt:List literal</a> containing a blank node whose identifier is NOT "b0" (<a href="#load-renaming-blank-example-simple-result"></a>):
+      <pre id="load-renaming-blank-example-simple"
+         title="Loading of a simple Composite Value with blank node"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      ex:s ex:p1 "[ _:b0 ]"^^cdt:List .
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-simple-result"
+         title="Loading of a simple Composite Value with blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p1> "[ _:b001 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>Evaluating a SPARQL Update in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in the addition of two triples to the dataset in which the newly-added triples' <a>cdt:List literal</a>s contain different blank nodes and where those blank node identifiers MUST be different and MUST NOT be "b0" (<a href="#load-renaming-blank-example-load-twice-result"></a>):</p>
```

Comment:
Ignoring the "NOT b0" language for the moment, proposed new language:

> Evaluating a SPARQL Update in Example 46 with two INSERT DATA operations with the same blank-node-containing cdt:List literal MUST result in a system-internal representation of two triples whose cdt:List literal objects' term lists contain different blank nodes and where the literals' blank node identifiers MUST be different and MUST NOT be "b0" (Example 47):

Reply:
That's better, but perhaps even more clear:

> Evaluating a SPARQL Update in Example 46 with two INSERT DATA operations with the same blank-node-containing cdt:List literal MUST result in a system-internal representation of two triples whose objects are cdt:List literals containing different blank nodes that, in the lexical form of these literals, are captured by different blank node identifiers (Example 47):

This would also remove the part about "NOT b0".

Reply:
I have added the edit suggestion above in a new comment (https://github.com/awslabs/SPARQL-CDTs/pull/2/files#r1584837148) from where you can apply it directly to the current version of his PR.